Change "validate" to "check"

klequis opened the pull request:
Fixes https://github.com/mdn/content/issues/11671

## Changes made

It appears that the script `setupTypeScript.js` has been changed so the script in `package.json` formally named 'validate' is now named 'check'. Personally, I like the word validate better but am proposing changes to the tutorial that reflect the script as it currently is.

Code from current script. Starts on line 33.
```js
// Add script for checking
packageJSON.scripts = Object.assign(packageJSON.scripts, {
  "check": "svelte-check --tsconfig ./tsconfig.json"
})
```



## Other Possible Changes

There are a couple of place where I was unsure if the word "validate" should be changed to "check".

**bold emphasis added just temporary to highlight the difference.**

Don't know if you would want to change the word "validate" to "check" in this context as `check` is doing validation.

(1)
```
"Now we will undo these changes in order to start porting our application to TypeScript, so we won't be bothered with all the **validate** warnings." 
```
_(or maybe say "validation warnings" to avoid confusion)_

It could be changed to:

```
"Now we will undo these changes in order to start porting our application to TypeScript, so we won't be bothered with all the **check** warnings."
```

---
(2)

`validate` seems to mean something different here but wanted to double check that.

"Now if we try to create a `localStore` with something that cannot be converted to JSON via `JSON.stringify()`, for example an object with a function as a property, VS Code/`validate` will complain about it:"

--

I'm happy to make the above changes if desired.

<!-- 👀 Thanks for opening a PR! Read comments like this one to get your PR merged faster. -->
#### Summary
<!-- ✍️ In a sentence or two, describe your changes -->

#### Motivation
<!-- ❓ Why are you making this change? Help us understand how your changes help readers. -->
`npm run validate` is now `npm run check`

#### Supporting details
<!-- 🔗 Link to supporting information, such as bug trackers, source control, release notes, and vendor docs. -->

Line #33 in the below script creates a script named 'check' not 'validate'.
```
https://github.com/sveltejs/template/blob/master/scripts/setupTypeScript.js
```

#### Related issues
<!-- 🔨 If applicable, use "Fixes #XYZ" -->

#### Metadata
<!-- ✅ Check a box if applicable, like this: [x]

This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [X ] Fixes a typo, bug, or other error

<!-- 👷‍♀️ After submitting, review the results of the "Checks" tab! -->

klequis committed:
Change "validate" to "check"

## Changes made

It appears that the script `setupTypeScript.js` has been changed so the script in `package.json` formally named 'validate' is now named 'check'. Personally, I like the word validate better but am proposing changes to the tutorial that reflect the script as it currently is.

Code from current script. Starts on line 33.
```js
// Add script for checking
packageJSON.scripts = Object.assign(packageJSON.scripts, {
  "check": "svelte-check --tsconfig ./tsconfig.json"
})
```



## Other Possible Changes

There are a couple of place where I was unsure if the word "validate" should be changed to "check".

**bold emphasis added just temporary to highlight the difference.**

Don't know if you would want to change the word "validate" to "check" in this context as `check` is doing validation.

(1)
```
"Now we will undo these changes in order to start porting our application to TypeScript, so we won't be bothered with all the **validate** warnings." 
```
_(or maybe say "validation warnings" to avoid confusion)_

It could be changed to:

```
"Now we will undo these changes in order to start porting our application to TypeScript, so we won't be bothered with all the **check** warnings."
```

---
(2)

`validate` seems to mean something different here but wanted to double check that.

"Now if we try to create a `localStore` with something that cannot be converted to JSON via `JSON.stringify()`, for example an object with a function as a property, VS Code/`validate` will complain about it:"

--

I'm happy to make the above changes if desired.

Rumyra commented:
Thanks so much for your contribution @klequis !

For (1) I would say `"Now we will undo these changes in order to start porting our application to TypeScript, so we won't be bothered with all the **check** warnings."`

And for (2) yes this is fine as it is 👍

klequis committed:
line 277: Change validate to check

On line 277 changed the word 'validate' to 'check'

**from:** Now we will undo these changes in order to start porting our application to TypeScript, so we won't be bothered with all the validate warnings.

**to:** Now we will undo these changes in order to start porting our application to TypeScript, so we won't be bothered with all the check warnings.

AnilSeervi reviewed (commented):
Fixes #11671

Rumyra commented:
This looks great @klequis - thank you!
